jwst/regtest/test_associations_standards.py

```diff
@@ -90,7 +90,7 @@ def __init__(
     MakePars('pool_030_mir_lrs_nods_bkg'),
     MakePars('pool_031_mir_lrs_nonod_bkg'),
     MakePars('pool_032_nircam_wfss'),
-    MakePars('pool_034_wfss_parallel'),
+    MakePars('pool_034_wfss_outlier_percentarallel'),
```

Comment:
@emolter - this change looks like it was probably unintentional

Reply:
thanks for catching, will fix